flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Is that because of ITK?

Reply:
No, because of https://github.com/flathub/flathub/pull/2679#discussion_r771250560
See also https://invent.kde.org/packaging/flatpak-kde-runtime/-/issues/19